x-pack/plugins/cases/server/client/cases/update.test.ts

```diff
@@ -994,13 +1039,47 @@ describe('update', () => {
               },
             ],
           },
-          clientArgs
+          clientArgs,
+          casesClient
         )
       ).rejects.toThrow('Error: Invalid duplicated custom field keys in request: duplicated_key');
     });
+
+    it('throws when customFields keys are not present in configuration', async () => {
```

Comment:
It seems that we test it the same below.

Reply:
Slightly different, the previous test has has duplicated keys.
That throws `'Error: Invalid duplicated custom field keys in request: duplicated_key'`.

This one is `'Error: The case with case id mock-id-1 has the following invalid custom field keys: missing_key'`.

Reply:
I see. Thanks!